sebs/azure/cli.py

```
@@ -1,7 +1,38 @@
+    result = cli.execute("az group list")
+    
+    # Upload function package
+    cli.upload_package(local_dir, container_dest)
+    ```
+"""
+
 import io
 import logging
 import os
 import tarfile
+from typing import Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)